### ISSUE TITLE
Integrating the new proxy feature

### ISSUE THREAD

**Issue by cagataygurturk** [lambadaframework/lambadaframework#35]
API Gateway now has a new feature:
https://docs.aws.amazon.com/apigateway/latest/developerguide/api-gateway-set-up-simple-proxy.html

It is leaving the vast majority of lambada framework obsolete, like method creating logic etc.

Also it'll automatically resolve issues like https://github.com/lambadaframework/lambadaframework/issues/12 https://github.com/lambadaframework/lambadaframework/issues/11

It needs a big refactor but the business value is very big, so it should be implemented as soon as possible.

**Comment by arran4** [comment]
It removes a significant portion of the code base right?

**Comment by cagataygurturk** [comment]
That's correct. And it means we will be able to support more features with less code.

**Comment by davidmco65** [comment]
Any progress on this?

**Comment by arranubels** [comment]
Can't figure out how to programmatically create the lambda proxy.

**Comment by cagataygurturk** [comment]
Of course they also did not add Cloudformation support for that.

**Comment by arranubels** [comment]
So where does that leave us?

**Comment by arranubels** [comment]
Actually my AWS SDK is old.. Checking .60

**Comment by arranubels** [comment]
Got it.

**Comment by atramos** [comment]
I think this is also a dependency for resolving Issue #57

**Comment by refacktor** [comment]
I've been experimenting with this and got a working branch in https://github.com/atramos/serverless-jax-rs/commits/proxy-redux, unfortunately I found that it doesn't really help much and actually makes it harder to fix #57.
